test/fixtures/users.yml

```diff
@@ -0,0 +1,11 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+one:
+  name: MyString
+  email: MyString
+  about: MyString
+
+two:
+  name: MyString
+  email: MyString
+  about: MyString
```

Comment:
We do not need this fixtures, since we are decided to use Rspec. Remove this file.

Reply:
We will use factory bot for this proposes https://github.com/thoughtbot/factory_bot